lib/cli.rs

```diff
@@ -86,6 +86,27 @@ pub async fn put_deploy(
         .map_err(CliError::from)
 }
 
+/// Speculatively creates a [`Deploy`] and sends it to the network for execution.
```

Comment:
I think the term "speculative" applies exclusively to the execution of the deploy on the node receiving it.  It follows quite a different code path through the node, and a bunch of caveats are implied in any successful response.

So while I don't think this is the place to try and describe the node's process, I guess I'd rephrase this to something like:
```suggestion
/// Creates a [`Deploy`] and sends it to the specified node for speculative execution.
```

Reply:
Done.